### DISCUSSION TITLE
[closed][manuscript]Idea found in schema comments: Registry of workflow and parameter settings.

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#124] [Team Posts]
"(We suggest to create a registry of workflows and parameter settings that one can point to - ELIXIR follow-up project?)"

Should we include this idea in the manuscript?

**Comment by dzerbino** [comment]
We should make sure nobody else is working on this at the moment (highly likely)

**Comment by sveinugu** [comment]
https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/41 links to this. Closing
